pom.xml

```diff
@@ -80,6 +75,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.hamcrest</groupId>
```

Comment:
LambdaJ had transitive dependency `org.hamcrest:hamcrest-all:1.1` which was used implicitly in tests